prototype_source/lite_interpreter.rst

```diff
@@ -28,51 +32,31 @@ Get ImageSegmentation demo app in Android: https://github.com/pytorch/android-de
-   + xargs ls -lah
-   -rw-r--r--  1 chenlai  staff    13M Feb 11 11:48 /Users/chenlai/pytorch/android/pytorch_android/build/outputs/aar/pytorch_android-release.aar
-   -rw-r--r--  1 chenlai  staff    36K Feb  9 16:45 /Users/chenlai/pytorch/android/pytorch_android_torchvision/build/outputs/aar/pytorch_android_torchvision-release.aar
-
-3. **Use the PyTorch Android libraries built from source in the ImageSegmentation app**: Create a folder `libs` in the path, the path from repository root will be `ImageSegmentation/app/libs`. Copy `pytorch_android-release` to the path ``ImageSegmentation/app/libs/pytorch_android-release.aar``. Copy `pytorch_android_torchvision` (downloaded from `Pytorch Android Torchvision Nightly <https://oss.sonatype.org/#nexus-search;quick~torchvision_android/>`_) to the path ``ImageSegmentation/app/libs/pytorch_android_torchvision.aar``. Update the `dependencies` part of ``ImageSegmentation/app/build.gradle`` to
+2. **Use the PyTorch Android library in the ImageSegmentation app**: Update the `dependencies` part of ``ImageSegmentation/app/build.gradle`` to
 
 .. code:: gradle
 
-   dependencies {
-       implementation 'androidx.appcompat:appcompat:1.2.0'
-       implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
-       testImplementation 'junit:junit:4.12'
-       androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-       androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
-
-
-       implementation(name:'pytorch_android-release', ext:'aar')
-       implementation(name:'pytorch_android_torchvision', ext:'aar')
-
-       implementation 'com.android.support:appcompat-v7:28.0.0'
-       implementation 'com.facebook.fbjni:fbjni-java-only:0.0.3'
-   }
-
-Update `all projects` part in ``ImageSegmentation/build.gradle`` to
+    repositories {
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots"
+        }
+    }
 
+    dependencies {
+        implementation fileTree(dir: "libs", include: ["*.jar"])
+        implementation 'androidx.appcompat:appcompat:1.2.0'
+        implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
+        testImplementation 'junit:junit:4.12'
+        androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+        androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+        implementation 'org.pytorch:pytorch_android:1.9.0-SNAPSHOT'
+        implementation 'org.pytorch:pytorch_android_torchvision:1.9.0-SNAPSHOT'
+
+        implementation 'com.android.support:appcompat-v7:28.0.0'
+        implementation 'com.facebook.fbjni:fbjni-java-only:0.0.3'
+    }
 
-.. code:: gradle
 
-    allprojects {
-        repositories {
-            google()
-            jcenter()
-            flatDir {
-                dirs 'libs'
-            }
-        }
-    }
 
 4. **Update model loader api**: Update ``ImageSegmentation/app/src/main/java/org/pytorch/imagesegmentation/MainActivity.java`` by
```

Comment:
No more step 3?

Reply:
After 1.9, no more step 3

Reply:
Good catch, it should be step 3 now

Reply:
Fix in latest commit. Also update the name `pytorch_android-release.aar`